nodes.adoc

```diff
@@ -0,0 +1,1163 @@
+= Nodes
+
+toc::[]
+
+
+## Configuring your cluster to place pods on overcommitted nodes
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f cro-namespace.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f cro-og.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f cro-sub.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f cro-cr.yaml
+----
+
+
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=small-pods
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Configuring an {product-title} cluster for pods
+
+[source,terminal]
+----
+$ oc create -f <file_or_dir_path>
+----
+
+
+[source,terminal]
+----
+$ oc create -f </path/to/file> -n <project_name>
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Configuring cluster memory to meet container memory and risk requirements
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+## Estimating the number of pods your {product-title} nodes can hold
+
+[source,terminal]
+----
+$ oc create sa cluster-capacity-sa
+----
+
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user cluster-capacity-role \
```

Comment:
Requires a declarative alternative

Reply:
@siamaksade Do you know if the declarative for this command is a ClusterRoleBinding?

Reply:
Yes that is correct.